apps/hash-graph/bin/cli/src/subcommand/type_fetcher.rs

```diff
@@ -58,3 +80,15 @@ pub async fn type_fetcher(args: TypeFetcherArgs) -> Result<(), GraphError> {
 
     Ok(())
 }
+
+async fn healthcheck(address: TypeFetcherAddress) -> Result<(), GraphError> {
+    let transport = tarpc::serde_transport::tcp::connect(
```

Comment:
Does this `connect` call timeout or spin/wait indefintely until a connection is made? Their docs don't mention anything regarding this

Reply:
We'd have timeouts in the hosted instance as well to prevent this from passing, so we should be good. We can come back to this if we see problems with indefinite waiting anywhere and add a timeout

Reply:
I approved just now, but we decided we may as well add the timeout now